python/mach_bootstrap.py

```diff
@@ -111,7 +110,11 @@ def _activate_virtualenv(topdir):
         except (subprocess.CalledProcessError, OSError):
             sys.exit("Python virtualenv failed to execute properly.")
 
-    execfile(activate_path, dict(__file__=quote(activate_path)))
+    execfile(activate_path, dict(__file__=activate_path))
+
+    python = find_executable("python")
+    if python is None or not python.startswith(virtualenv_path):
+        sys.exit("Python virtualenv failed to activate.")
```

Comment:
@frewsxcv I appreciate you taking the time to test this. Could you please include these 3 lines in your test as well? It might shine some light on the problem.